modules/dxp/apps/portal-search-tuning/portal-search-tuning-rankings-web/src/main/java/com/liferay/portal/search/tuning/rankings/web/internal/constants/ResultRankingsConstants.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.portal.search.tuning.rankings.web.internal.constants;
+
+/**
+ * @author Olivia Yu
+ */
+public class ResultRankingsConstants {
+
+	public static final int STATUS_ACTIVE = 0;
+
+	public static final int STATUS_INACTIVE = 1;
+
+	public static final String STATUS_LABEL_ACTIVE = "active";
+
+	public static final String STATUS_LABEL_INACTIVE = "inactive";
+
+	public static String getStatusLabel(int status) {
+		if (status == STATUS_ACTIVE) {
+			return STATUS_LABEL_ACTIVE;
+		}
+		else if (status == STATUS_INACTIVE) {
+			return STATUS_LABEL_INACTIVE;
+		}
+
+		return null;
+	}
+
+}
```

Comment:
Great job with this! 👏 

It might make more sense making active = 1 and inactive = 0 though?

Reply:
Also instead of returning `null` let's return `StringPool.BLANK` since a String is expected to return and could cause issues when attempting to display a null value.